.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
```

Comment:
You seem to have the same templates in two places... in `.github` and in `github-templates`... is that right?